.github/ISSUE_TEMPLATE/Proposal.md

```diff
@@ -0,0 +1,15 @@
+---
+name: 🤔 Proposal
+about: Have a nice proposal?
+---
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | -----
+| Version     | x.y.z
```

Comment:
I don't think the version is relevant here, is it?

Reply:
Yes, you're right!